modules/oe_whitelabel_search/tests/src/Kernel/SearchBlockTest.php

```diff
@@ -122,10 +122,10 @@ public function testBlockRendering(): void {
     $render = $this->container->get('renderer')->renderRoot($build);
```

Comment:
I was going to rename the variables here, but I then decided to rather not touch this part.
E.g. `$render` -> `$markup`, `$entity` -> `$block_entity`.
We can do this another day.